addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -99,14 +98,14 @@ def portal_my_opportunities(self, page=1, date_begin=None, date_end=None, sortby
         domain = self.get_domain_my_opp(request.env.user)
 
         today = fields.Date.today()
-        this_week_end_date = fields.Date.to_string(fields.Date.from_string(today) + datetime.timedelta(days=7))
+        current_user_activities = CrmLead.search([('activity_user_id', '=', request.env.user.id)])
```

Comment:
@nihp-odoo  Name should be related to `leads`.

Reply:
Hello @ibo-odoo ,
Requested changes are done. Please have a look.
Thanks.

Reply:
@nihp-odoo one last comment - improve the commit message.